s3/pkg/datastore/ceph/ceph.go

```diff
@@ -319,7 +336,42 @@ func (ad *CephAdapter) ListParts(ctx context.Context, multipartUpload *pb.ListPa
 }
 
 func (ad *CephAdapter) BackendCheck(ctx context.Context, backendDetail *pb.BackendDetailS3) error {
-    return ErrNotImplemented
+	randId := uuid.NewV4().String()
+	input := &pb.Bucket{
+		Name: sampleBucket + randId,
+	}
+	for i := 1; i <= 3; i++ {
```

Comment:
What is the need to put retry in bucket create? Any scenario you see?

Reply:
yes if there is some issue due to network it will get resolved in 2nd  or maybe third call